frontend/src/lib/components/Papercups.tsx

```diff
@@ -17,7 +17,7 @@ export function Papercups(): JSX.Element {
             subtitle="Ask us anything in the chat window below 😊"
             newMessagePlaceholder="Start typing…"
             primaryColor="#5375ff"
-            greeting="Hi! Send us a message and we'll respond as soon as we can."
+            greeting="We'll respond as soon as we can. You can also check posthog.com/docs if you need assistance."
```

Comment:
Think maybe we want to keep the greeting? i.e. "Hi! We'll respond ...", also we could highlight the benefit of checking the docs which is getting an answer right away, so something like: "Hi! We'll respond as soon as we can. For an immediate answer you may want to search our docs (posthog.com/docs)"

Reply:
Yep, made a quick tweak to it so that it's staying as short as possible.

Reply:
Made an additional update by adding in an 'awayMessage' field after discussion with Alex at PC